roles/client_side_tests/tasks/test_e2e.yml

```diff
@@ -0,0 +1,112 @@
+---
+- name: "Extract URL from stf-connectors template"
+  shell: grep interconnect /home/stack/virt/stf-connectors* | sed 's;.*\(-service-telemetry.*\);default-prometheus-proxy\1;'
```

Comment:
@lnatapov, does this work to get the same info? 
    oc get route/default-prometheus-proxy -ojson | jq '.spec.host'

If so, I can update this to allow passing in the var, and add a when: to this task to run only if it isn't already defined.

Reply:
@elfiesmelfie yes. if you run the following command it will give you the same result but it will give you the promethues URL with double quotes.

Example:
[quicklab@upi-0 ~]$ oc get route/default-prometheus-proxy -ojson | jq '.spec.host'
"default-prometheus-proxy-service-telemetry.apps.leonidcluster.lab.upshift.rdu2.redhat.com"

In order to remove the quotes and get clean URL run this:

[quicklab@upi-0 ~]$ oc get route/default-prometheus-proxy -ojson | jq '.spec.host' |  sed 's/^"\|"$//g'
default-prometheus-proxy-service-telemetry.apps.leonidcluster.lab.upshift.rdu2.redhat.com